packages/edit-site/src/plugins/index.js

```diff
@@ -17,6 +20,22 @@ registerPlugin( 'edit-site', {
 				<ToolsMoreMenuGroup>
 					<SiteExport />
 					<WelcomeGuideMenuItem />
+					<MenuItem
+						icon={ external }
```

Comment:
```suggestion
						icon={ external }
						role="menuitem"
```

Reply:
I omitted this since its default value - https://developer.wordpress.org/block-editor/reference-guides/components/menu-item/#role. But have no problem with the role specified as well :)

Reply:
I missed that, sorry. You can revert this commit if you want.

Reply:
There's no need to revert. Just wanted to clarify :)